### PR TITLE
Feature/encoding desc parser

### PR DESCRIPTION
**Pull request by ChiaraDipi**

### CODE REVIEW

File: src/app/models/evt-models.ts [hunk 1]
```diff
@@ -849,3 +849,129 @@ export class SourceDesc extends GenericElement {
...
+    unitDecl: Array<ParseResult<GenericElement>>; // TODO: Add specific type when unitDecl is handled
+    schemaSpec: Array<ParseResult<GenericElement>>; // TODO: Add specific type when schemaSpec is handled
+    schemaRef: Array<ParseResult<GenericElement>>; // TODO: Add specific type when schemaRef is handled
+}
+
+export class ProjectDesc extends GenericElement {
+    content: Paragraph[];
+}
+
+export class SamplingDecl extends GenericElement {
```
Comment (szenzaro) [inline src/app/models/evt-models.ts:872]: Use type alias for this

File: src/app/models/evt-models.ts [hunk 2]
```diff
@@ -849,3 +849,129 @@ export class SourceDesc extends GenericElement {
...
+    content: Paragraph[];
+}
+
+export type CorrectionStatus = 'high' | 'medium' | 'low' | 'unknown';
+export type CorrectionMethod = 'silent' | 'markup';
+export class Correction extends GenericElement {
```
Comment (szenzaro) [inline src/app/models/evt-models.ts:878]: Extend `ProjectDesc`?

File: src/app/models/evt-models.ts [hunk 3]
```diff
@@ -849,3 +849,129 @@ export class SourceDesc extends GenericElement {
...
+    content: Paragraph[];
+    status?: CorrectionStatus;
+    method?: CorrectionMethod;
+}
+
+export type NormalizationMethod = 'silent' | 'markup';
+export class Normalization extends GenericElement {
```
Comment (szenzaro) [inline src/app/models/evt-models.ts:885]: Extend `ProjectDesc`?

File: src/app/models/evt-models.ts [hunk 4]
```diff
@@ -849,3 +849,129 @@ export class SourceDesc extends GenericElement {
...
+
+export class Segmentation extends GenericElement {
+    content: Paragraph[];
+}
+
+export class StdVals extends GenericElement {
+    content: Paragraph[];
+}
+
+export class Interpretation extends GenericElement {
```
Comment (szenzaro) [inline src/app/models/evt-models.ts:919]: Type alias these

**Review by szenzaro — CHANGES_REQUESTED** [review]
Missing Changelog update

File: src/app/models/evt-models.ts (hunk 1 above)
Comment (ChiaraDipi) [reply src/app/models/evt-models.ts:872]: If I use type alias (`export type SamplingDecl = ProjectDesc;`), then I get an error in the parser in the setting of `type` property:
```
parse(xml: XMLElement): SamplingDecl {
    return {
      ...super.parse(xml),
      type: SamplingDecl,
      content: queryAndParseElements<Paragraph>(xml, 'p'),
    };
  }
```
The error says: «'SamplingDecl' only refers to a type, but is being used as a value here.»

File: src/app/models/evt-models.ts (hunk 4 above)
Comment (ChiaraDipi) [reply src/app/models/evt-models.ts:919]: See previous comment

**Review by szenzaro — APPROVED** [review]

**Pull request merged**